com.github.scrivano.scrivano.yml

```diff
@@ -0,0 +1,31 @@
+app-id: com.github.scrivano.scrivano
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: Scrivano
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=host
+  - --device=dri
+modules:
+  - name: Scrivano
+    buildsystem: simple
+    build-commands:
+    - install -D Scrivano /app/bin/Scrivano
+    - install -D libskia.so /app/lib/libskia.so
+    - install -D libpdfium.so /app/lib/libpdfium.so
+    - install -Dm644 com.github.scrivano.scrivano.appdata.xml -t /app/share/metainfo
+    - install -Dm644 com.github.scrivano.scrivano.desktop -t /app/share/applications
+    - install -Dm644 app_icon.svg /app/share/icons/hicolor/scalable/apps/com.github.scrivano.scrivano.svg
+    sources:
+    - type: archive
+      url: https://github.com/scrivanolabs/ScrivanoForLinux/releases/download/0.13.4/Scrivano_flatpak_files_0.13.4.zip
+      sha256: 48841ba44d95b6d7b625266c9d4622b0492465f29fec2491292139054b27cd09
+    - type: file
+      path: com.github.scrivano.scrivano.appdata.xml
+    - type: file
+      path: com.github.scrivano.scrivano.desktop
+    - type: file
+      path: app_icon.svg
```

Comment:
neitiher of these 3 files is available.

Reply:
Thanks for pointing this out, indeed those files are contained in my Github repo. Done.